src/dory/util/connect_to_host.cc

```diff
@@ -32,11 +32,18 @@ using namespace Socket;
 
 void Dory::Util::ConnectToHost(const char *host_name, in_port_t port,
     TFd &result_socket) {
+  int tmp_family; 
   assert(host_name);
   result_socket.Reset();
 
```

Comment:
You should be able to avoid adding `tmp_family` and the `strstr()` call by passing `AF_UNSPEC` rather than `AF_INET` to the `Db::TCursor` constructor call.  Can you give that a try and see if it works for you?  Thanks!

Reply:
yes, AF_UNSPEC works, learns a lot, thanks.